src/test/java/com/thealgorithms/dynamicprogramming/Sum_Of_SubsetTest.java

```diff
@@ -0,0 +1,21 @@
+package com.thealgorithms.dynamicprogramming;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+
+import org.junit.jupiter.api.Test;
+
+class Sum_Of_SubsetTest {
+
+    @Test
+    void basicCheck() {
+        assertEquals(true, Sum_Of_Subset.subsetSum(new int[] {7, 3, 2, 5, 8}, 4, 14));
+
+        assertEquals(true, Sum_Of_Subset.subsetSum(new int[] {4, 3, 2, 1}, 3, 5));
+
+        assertEquals(false, Sum_Of_Subset.subsetSum(new int[] {2, 5, 1, 6, 7}, 4, 4));
+
+        assertEquals(false, Sum_Of_Subset.subsetSum(new int[] {1, 7, 2, 9, 10}, 4, 14));
+
+        assertEquals(true, Sum_Of_Subset.subsetSum(new int[] {1, 7, 2, 9, 10}, 4, 13));
```

Comment:
Tests look good. You might consider _reordering_ them (unless there is some structure or logic behind the current order, which I do not see).